bin/find-windows-line-breaks

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Exists with code 1 if a `src` file contains Windows line breaks.
+
+for file in $( find src -type f ); do
+    grep -U $'\015' $file
```

Comment:
I made some changes to this file this morning that you might want to copy: https://github.com/partnermarketing/pm2-frontend/commit/ad1b5f633b2243791da42816b4d4a5eb7e9fb9d0